aspnet/SameSite/system-web-samesite.md

```diff
@@ -35,13 +74,31 @@ The November 19, 2019 updates for Windows updated .NET 4.7.2+ from the 2016 stan
 
 * Is **not** backwards compatible with the 2016 draft. For more information, see [Supporting older browsers](#sob) in this document.
 * Specifies cookies are treated as `SameSite=Lax` by default.
-* Specifies cookies that explicitly assert `SameSite=None` in order to enable cross-site delivery should be marked as `Secure`. `None` is a new entry to opt out.
+* Specifies cookies that explicitly assert `SameSite=None` in order to enable cross-site delivery should also be marked as `Secure`.
 * Is supported by patches issued as described in the KB's listed above.
 * Is scheduled to be enabled by [Chrome](https://chromestatus.com/feature/5088147346030592) by default in [Feb 2020](https://blog.chromium.org/2019/10/developers-get-ready-for-new.html). Browsers started moving to this standard in 2019.
 
+<a name="known"><a/>
+
+## Known Issues
+
+Because the 2016 and 2019 draft specifications are not compatible, the November 2019 .Net Framework update introduces some changes that may be breaking.
+
+1. Session State and Forms Authentication cookies are now written to the network as `Lax` instead of unspecified.
+    - While most applications will work with `SameSite=Lax` cookies, applications that POST across sites or applications that **make use
+    of `iframe`** may find that their session state or forms auth cookies are not being used as expected. To remedy this issue, change
+    the `cookieSameSite` value in the appropriate config section as discussed above.
```

Comment:
Isn't this broken at the moment?

Reply:
The recommendation would be to set the values to 'None', which is not broken at the moment. Only 'Unspecified' is troublesome.